app/controllers/api/v1/screenshots_controller.rb

```diff
@@ -27,6 +27,10 @@ def screenshot_params
         params.require(:screenshot).permit(:name, :locale_id, :product_version_id, images: [])
       end
 
+      def screenshots_json(screenshots)
+        screenshots.as_json.merge(Images: screenshots.images.map { |image| url_for(image) })
```

Comment:
can we use `images: ...` instead of `Images: ..`

Reply:
@rahulbajaj0509  Yes, we can. Shall I update it with "images:"?

Reply:
sure.